server/rest/src/main/java/org/infinispan/rest/resources/CacheResourceV2.java

```diff
@@ -548,6 +550,16 @@ private RestResponse getDetailResponse(Cache<?, ?> cache) {
          // Admin is needed
       }
 
+      Boolean rebalancingEnabled = null;
+      try {
+         if(cache instanceof AbstractDelegatingAdvancedCache) {
```

Comment:
I have a hunch this condition is not true for secured caches. It would be better if you used SecurityActions to extract the `LocalTopologyManager` via the component registry and invoke `isRebalancingEnabled()` on it.

Reply:
changing that